naxos_library_importer.user.js

```diff
@@ -0,0 +1,238 @@
+/* global $ jQuery MBImport */
+'use strict';
+// ==UserScript==
+// @name         Import Naxos Music Library releases to MusicBrainz
+// @namespace    https://github.com/murdos/musicbrainz-userscripts
+// @author       loujine
+// @version      2019.9.22
+// @downloadURL  https://raw.githubusercontent.com/murdos/musicbrainz-userscripts/master/naxos_library_importer.user.js
+// @updateURL    https://raw.githubusercontent.com/murdos/musicbrainz-userscripts/master/naxos_library_importer.user.js
+// @icon         https://raw.githubusercontent.com/murdos/musicbrainz-userscripts/master/assets/images/Musicbrainz_import_logo.png
+// @description  Add a button to import Naxos Music Library releases to MusicBrainz
+// @compatible   firefox+tampermonkey
+// @license      MIT
+// @include      http*://*naxosmusiclibrary.com/catalogue/item.asp*
+// @require      lib/mbimport.js
+// @require      lib/mbimportstyle.js
+// @grant        none
+// @run-at       document-end
+// ==/UserScript==
+
+// seems that $ is predefined but does not work
+$ = jQuery;
+
+var url = document.URL.split('.');
+url.splice(0, 1, 'https://www');
+const editNote = `
+    Imported from ${url}
```

Comment:
URL was incorrect during my test, I got `https://www,com/catalogue/item,asp?cid=ALPHA581`

Reply:
I don't quite understand how my code was supposed to work... :)
Anyway I fixed it